### PR TITLE
Feature/Create article model

### LINKED ISSUE #1: Create model for an article
An article has a title and a text field.

### PR DESCRIPTION
**Pull request by delenamalan**
Fix #1. Create an article model with title and text fields. Create a migration to create this model in the database. Register model on the admin view.

### CODE REVIEW

File: demo_project/demo_app/admin.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
```
Comment (hedleyroos) [inline demo_project/demo_app/admin.py]: See https://www.python.org/dev/peps/pep-0008/ for Python style guidelines.

File: demo_project/demo_app/admin.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 
-# Register your models here.
+from .models import Article
+
```
Comment (hedleyroos) [inline demo_project/demo_app/admin.py]: Here you need one more blank line. The rule is 2 lines between classes and functions (not methods), and 2 lines between imports and actual code.

File: demo_project/demo_app/models.py [hunk 3]
```diff
@@ -1,3 +1,9 @@
 from django.db import models
 
-# Create your models here.
+
+class Article(models.Model):
+    article_title = models.CharField(max_length=200)
```
Comment (hedleyroos) [inline demo_project/demo_app/models.py]: You don't have to prefix the fields with article_. It is already encapsulated in an Article class.

**Comment by AltusBarry** [comment]
+1

**Pull request merged**